lib/proforma/helpers/import_helpers.rb

```diff
@@ -64,12 +64,14 @@ def add_test_configuration(test, test_node)
       end
 
       def extra_configuration_from_test_configuration(test_configuration_node)
```

Comment:
These are also multiple "nodes", right?

```suggestion
      def extra_configuration_from_test_configuration(test_configuration_nodes)
```

Reply:
In this case it actually still is the test-configuration node
```XML
<test-configuration>
  <filerefs>
    <fileref refid="test_file_1"/>
  </filerefs>
  <unit:unittest framework="JUnit" version="4.10">
    <unit:entry-point>HelloWorldTest</unit:entry-point>
  </unit:unittest>
</test-configuration>
```
its children (the "nodes") get filtered and assigned for processing in the next line
```ruby
configuration_any_nodes = test_configuration_node.children....
```

Reply:
Ah, I see, makes sense -- thanks!